mta-sts.md

```diff
@@ -498,4 +498,34 @@ https://mta-sts.example.com/.well-known/mta-sts.json:
 
 ~~~~~~~~~
 
+# Appendix 2: Sending MTA State Machine
+
+It is helpful to think of the different states of a sending MTA and a receiving
```

Comment:
I also find it wordy, but given some of the feedback we've received from others about expanding various definitions, I don't think it is unnecessarily so.  I'd say that as long it makes sense (which it seems to do), we let it go like this.